src/App.js

```diff
@@ -1,10 +1,21 @@
+import { Route, Router, Routes } from "react-router-dom";
 import "./App.css";
+import Navbar from "./components/Navbar/Navbar";
 import logo from "./logo.png";
+import Home from "./pages/Home/Home";
+import { BrowserRouter } from "react-router-dom";
+import Footer from "./components/Footer/Footer";
```

Comment:
You can export all your components from a single `index.js` file, from that way you can import all your components from a single file.